media-tv/plex-media-server/plex-media-server-1.13.9.5456.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker user
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+COMMIT="ecd600442"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server"
+SRC_URI="
+	amd64? ( https://downloads.plex.tv/plex-media-server/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_amd64.deb )
+	x86? ( https://downloads.plex.tv/plex-media-server/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi"
+
+DEPEND="sys-apps/fix-gnustack"
+RDEPEND="avahi? ( "net-dns/avahi" )"
+
+EXECSTACKED_BINS=( "${ED%/}/usr/lib/plexmediaserver/libgnsdk_dsp.so*" )
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/${_APPNAME}/.*"
+	"usr/lib/${_APPNAME}/Resources/Python/lib/python2.7/.*"
+)
+
+S="${WORKDIR}"
+
+pkg_setup() {
+	enewgroup ${_USERNAME}
+	enewuser ${_USERNAME} -1 /bin/bash /var/lib/${_APPNAME} "${_USERNAME},video"
+}
+
+src_unpack() {
+	unpack_deb ${A}
+}
+
+src_prepare() {
+	eapply "${FILESDIR}"/plexmediamanager.desktop.patch
+	default
+}
+
+src_install() {
+	# Move the config to the correct place
+	local config_vanilla="/etc/default/"${_APPNAME}""
+	local config_path="/etc/${_SHORTNAME}"
+	dodir "${config_path}"
+	insinto "${config_path}"
+	doins "${confing_vanilla#/}"
+	sed -e "s#${config_vanilla}#${config_path}/$(basename "${CONFIG_VANILLA}")#g" \
```

Comment:
Fix `${CONFIG_VANILLA}` to lowercase here as well. I'm not sure if basename needs to die, but since its a subcommand for sed, I doubt it.